aws/table_aws_wellarchitected_answer.go

```diff
@@ -0,0 +1,317 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedAnswer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_answer",
+		Description: "AWS Well-Architected Answer",
+		Get: &plugin.GetConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "lens_alias", Require: plugin.Required},
+				{Name: "question_id", Require: plugin.Required},
+				{Name: "workload_id", Require: plugin.Required},
+			},
```

Comment:
@karanpopat Should we not add `IgnoreConfig`? I believe `IgnoreConfig` will work in the get config, not in the list config, if we use `ParentHydrate`. If not, could you please add a comment on why we have removed the `IgnoreConfig`?